docs/publishing.md

```diff
@@ -1,96 +1,126 @@
+packages) and then run `npm install`.
+  - ** Important ** : Don't run `npm install` in the Design System before you have unlinked the symlink; linking the directories effectively permits changes made in the Design System to take effect in `govuk-frontend`.
+  - Note: * This process only verifies that the current packages work, it doesn't
+  mean that all of the changes have been copied. *
+
+7. At this point, run `git status` to make sure no changes have been introduced
+since the commit in step 5. (Any contents of the `packages` folder will get pushed to npm in step 11.)
 
-6. In the console, log in to `npm` as user who has permission to publish to the `govuk-frontend` scope. You can use `npm whoami` to check your current user.
+8. In the console, log in to npm as user who has permission to publish to the `govuk-frontend` scope. You can use `npm whoami` to check your current user.
 
-7. Run
+9. Run
 ```
 lerna bootstrap
 ```
-You should see `lerna success Bootstrapped [number of] packages`. This will install all package dependencies and link any cross-dependencies.
+You should see `lerna success Bootstrapped [number of] packages`. This will
+install all package dependencies and link any cross-dependencies.
 
-8. Run
+10. Run
 ```
 lerna updated
 ```
 This will display the packages that have changed.
 
-9. Run
+11. Run
 ```
 lerna publish -m "chore(release): update packages and publish"
 ```
-Here, Lerna will prompt you to select the new version for each package. If you select
-`minor`/`major` etc., Lerna will complete the version number for you. In private beta, we have selected `custom` and specified the new version number manually for each package
-(see below):
+Here, Lerna will prompt you to select the new version for each package. If you
+select `minor`/`major` etc., Lerna will complete the version number for you. In
+private beta, we have selected `custom` and specified the new version number
+manually for each package (see below). Also see step 3 regarding the versioning of new components:
 ![Select version in Lerna](./img/lerna-select-version.png)
 
-10. Once you have updated all the package versions, confirm you want to publish the changes (see below).
+12. Once you have updated all the package versions, confirm you want to publish
+the changes (see below).
 ![Confirm publishing of changes in Lerna](./img/lerna-confirm-publish.png)
 
-11. If your changes included new packages, you need to grant the test user access to them:
+13. If your changes included new packages, you need to grant npm test user access to them:
 ```
-npm access grant read-only govuk-frontend:test @govuk-frontend/button`
+npm access grant read-only govuk-frontend:test @govuk-frontend/warning-text
 ```
 If you removed any packages, revoke the test user's access to them:
 ```
-npm access revoke govuk-frontend:test @govuk-frontend/button`
+npm access revoke govuk-frontend:test @govuk-frontend/link
```

Comment:
There's issues with this which we discovered today – anyone who's using an old version of Frontend can no longer deploy as they're reliant on packages they can no longer see.

I think realistically we'll need to continue to keep old packages around going forward. (Though we should probably clean up old packages we know no-one's using now, before people outside of the team start using it)

Reply:
shall be remove this line then?